core/server/data/migrations/versions/3.22/02-settings-key-renames.js

```diff
@@ -12,6 +12,17 @@ const renameMapping = [{
 }, {
     from: 'ghost_foot',
     to: 'codeinjection_foot'
+}, {
+    from: 'brand',
+    to: 'accent_color',
+    getValue: (oldValue) => {
```

Comment:
NBD but I'd go with `getFromValue` and `getToValue` naming here to keep it consistent with from/to convention.

Reply:
Done :relaxed: